docs/content/guides/developer/getting-started/upgrade-sui.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: Upgrade Sui
+description: placeholder
+keywords: [ placeholder ]
+---
+
+## Upgrade from Cargo
```

Comment:
I forgot about this topic. It should just go in the install topics, I think. It's already there, but it just needs to state that you use the same command to just install over it.